docs/user/DockerUsage.md

```diff
@@ -12,6 +12,8 @@ Run the following command to create the `viya4-deployment` Docker image using th
 
 ```bash
 docker build -t viya4-deployment .
+#or
+sudo docker build -t viya4-deployment .
```

Comment:
If you've setup docker correctly you do not need `sudo`. This needs to be removed.

Reply:
The only alternative to run sudo docker on Fedora 34, I found, is to use podman which did not work well for me.
So, I am telling from my own experience but if you point me to specific way to make non-root docker work on Fedora as generic Linux OS I will include it in some reference doc.

Reply:
I think I saw that doc and tried but it was failing on the networking side for me.
So I just did what was faster and easier to make it work.
I think this is legit alternative for not so savvy docker users as it ultimately worked for me.

Reply:
Yeah, that's going to keep this from merging no `sudo` to run docker. You'll need to get it to work without it.

Reply:
I, then, suggest to add this link 
https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user
to some part of your documentation.

Reply:
I also think it is worth mentioning using podman as alternative to docker as non-root. I had almost success with it without doing much of OS reconfiguration.